Makefile

```diff
@@ -372,6 +372,27 @@ verify-gen: generate ## Verify generated files
 vet: ## Runs the Go vet command on this project
 	go vet ./...
 
+## --------------------------------------
+## Tilt / Kind
+## --------------------------------------
+
+.PHONY: kind-create
+kind-create: ## create capz kind cluster if needed
+	./scripts/kind-with-registry.sh
+
+.PHONY: tilt-up
+tilt-up: kind-create ## start tilt and build kind cluster if needed
+	tilt up
+
+.PHONY: delete-cluster
+delete-cluster: delete-workload-cluster  ## Deletes the example kind cluster "capdo"
+	kind delete cluster --name=capdo
```

Comment:
If this target deletes the cluster created by `kind-create`, I think we should respect the `KIND_CLUSTER_NAME` environment variable which is used in the script. The same applies for the `kind-reset` target.

Reply:
thanks for your review @xmudrii 
make the requested changes

Reply:
I see you added the `KIND_CLUSTER_NAME` variable, but should we use it as a name instead of `capdo` (e.g. `--name=$(KIND_CLUSTER_NAME)`?

Reply:
forgot that :( 🤦

Reply:
now it is done